readme.txt

```diff
@@ -104,6 +104,10 @@ If you have more questions you can always head over to https://wordpress.org/sup
 
 == Changelog ==
 
+= 1.1.16 =
+* Fixed the Board Member role not being available on subsites of a multisite if the plugin was network activated.
```

Comment:
It looks like there might be a word missing after "multisite" here.

Reply:
@jg314 what word are you thinking would go there?

Reply:
Sorry I wasn't clear on this one. I was thinking it would be "subsites of a multisite installation". That's normally how I see it written, but modify if you think there is something better or it's best as is.

Reply:
That makes sense. This is updated.